api-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  db:
+    image: postgres:14.0
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: password
```

Comment:
there's a number of config and app secrets committed in code, and we should consider secure provisioning

Reply:
Agree. This is for local dev. We're hardcoding some secrets in others parts of the code too. I'll create an issue for it